packages/phoenix-ng/projects/phoenix-ui-components/lib/components/phoenix-menu/config/config-slider/config-slider.component.ts

```diff
@@ -8,12 +9,36 @@ import { Component, Input, Output, EventEmitter } from '@angular/core';
 export class ConfigSliderComponent {
   @Input() value: number = 0;
   @Input() min: number = 0;
+  oldMin: number = 0;
   @Input() max: number = 100;
+  oldMax: number = 0;
   @Input() step: number = 1;
   @Input() allowCustomValue: boolean = false;
   @Output() onChange: EventEmitter<number> = new EventEmitter<number>();
 
   onValueChange(value: number) {
     value && this.onChange.emit(value);
   }
+
+  toggleMinCut(change: MatCheckboxChange) {
+    const value = change.checked;
+    if (value) {
+      this.min = this.oldMin;
+    } else {
+      this.oldMin = this.min;
+      this.min = Number.MIN_SAFE_INTEGER;
+    }
+    // We should probably disable the min input too, and change text to be infinity symbol or something?
+  }
+
+  toggleMaxCut(change: MatCheckboxChange) {
+    const value = change.checked;
+    if (value) {
+      this.max = this.oldMax;
+    } else {
+      this.oldMax = this.max;
+      this.max = Number.MIN_SAFE_INTEGER;
```

Comment:
This should be `Number.MAX_SAFE_INTEGER`?

I am not sure if this is the source of the behavior you mentioned.

Reply:
I don't think so - I don't think this code is ever called (I will remove it).